user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -331,3 +333,5 @@ Bugs Fixed
 **********
 
 - Fixed a bug when all types of ``Prepared Queries`` were returning a ``Result`` object instead of a bool value for write-type queries.
+- Fixed a bug when not always all variables in a JSON request were filtered if filters were applied with ``IncomingRequest::getVar()`` or ``IncomingRequest::getJsonVar()`` methods.
```

Comment:
```suggestion
- Fixed a bug with variable filtering in JSON requests ysing with ``IncomingRequest::getVar()`` or ``IncomingRequest::getJsonVar()`` methods.
```

Reply:
What's *ysing* ?

Reply:
Thanks. Fixed.

Reply:
Haha oops! Sorry for the typo, yes that was supposed to be "using" (and drop the "with"). Looks good.